src/main/java/org/apache/maven/model/jdom/util/JDomUtils.java

```diff
@@ -61,14 +62,29 @@ public static void addElement(Element element, Element root) {
    */
   public static void addElement(Element element, Element root, int index) {
     root.addContent(
-            index == -1 ? root.getContentSize() - 1 : index,
+            index,
             asList(
                     new Text("\n" + detectIndentation(root)),
                     element));
     resetIndentations(root, detectIndentation(root));
     resetIndentations(element, detectIndentation(root) + "  ");
   }
 
+  public static int getElementIndex(Element element, Element root) {
+    return root.indexOf(element);
+  }
+
+  private static int getLastElementIndex(Element root) {
+    List<Element> elements = root.getContent(new ElementFilter());
+
+    int size = elements.size();
+    if (size == 0) {
+      return -1;
+    } else {
```

Comment:
`else` could be dropped. Just return `return root.indexOf(elements.get(size - 1));`

Reply:
Still used to another coding style. In the past, I always added the else branches.